scripts/anchorage_events.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -e
```

Comment:
Remove the `set -e` you are already checking the status and exiting in case there is an error. If you use `set -e` you won't read the `echo "ERROR MESSAGE` you have added below.